common.js

```diff
@@ -1728,6 +1728,586 @@ const allSoloCars = {
       'all': ['ssp'],
     },
   },
+  'Nissan': {
+    '200SX': {
+      'all': ['hs'],
+      '1976': ['fsp'],
+      '1977': ['fsp'],
+      '1978': ['fsp'],
+      '1979': ['fsp'],
+      '1980': ['fsp'],
+      '1981': ['fsp'],
+      '1982': ['fsp'],
+      '1983': ['fsp'],
+      '1984': ['fsp'],
+      '1985': ['fsp'],
+      '1986': ['fsp'],
+      '1987': ['fsp'],
+      '1988': ['fsp']
+    },
+    '200SX SE-R': {
+      'all': ['fsp'],
+      '1995': ['sts'],
+      '1996': ['sts'],
+      '1997': ['sts'],
+      '1998': ['sts']
+    },
+    '200SX Turbo': {
+      'all': ['dsp']
+    },
+    '200SX (S10 chassis) - Alternate cylinder head: 11041-22010, 11041-U0600-A, 11041-U0602-SV, 11041-21901, or 11041-N7120': {
+      '1977': ['dp'],
+      '1978': ['dp'],
+      '1979': ['dp']
+    },
+    '200SX (S110 chassis) - Alternate cylinder head: 11041-22010, 11041-U0600-A, 11041-U0602-SV, 1041-21901, or 11041-N7120 Alternate engine: L20B or NAPS-Z': {
+      '1980': ['dp'],
+      '1981': ['dp'],
+      '1982': ['dp'],
+      '1983': ['dp']
+    },
+    '200SX (S12 chassis) - Alternate cylinder head: 11041-N7120. Engine: L20B or NAPS-Z': {
+      '1984': ['dp'],
+      '1985': ['dp'],
+      '1986': ['dp'],
+      '1987': ['dp'],
+      '1988': ['dp']
+    },
+    '200SX (B14 chassis)': {
+      '1995': ['ep'],
+      '1996': ['ep'],
+      '1997': ['ep'],
+      '1998': ['ep'],
+      '1999': ['ep']
+    },
+    '210': {
+      'all': ['fsp']
+    },
+    '210 (1397 & 1488 cc)': {
```

Comment:
Remove this and add `dp` to the existing `210` model, thanks!

Reply:
Addressed in the following commit:
https://github.com/Bjorn248/scca_classifier/pull/81/commits/12fe594c3581614af605b5623aaacf3781f3e4e5